### ISSUE TITLE
app specific fields should be removed

### ISSUE THREAD

**Issue by tschady** [finos-voice/voice-metadata-standard#12]
e.g. `buttonName` refers to the invoking application, and not the actual call metadata, and should be removed.

Vendors should be able to extend the format to include optional payloads.

**Comment by kemmiso** [comment]
Within comChain we leveraged AdditionalProperty for this type of payload. https://schema.org/additionalProperty

**Comment by leop-tcto** [comment]
Agreed about button name, however we need to define what the minimum metadata items are.

**Comment by vitorespindola** [comment]
I suggest to rename it to `contactName` or other more generic name where we can store a human friendly name for this callRecord when available, and keep this field as part of the standard as a non required field.

With this generic name we can store the name if it comes from any of these options:

* from the line itself. eg. contact name in the SIP From header
* the application. eg. the button name from c9 or any application specific identification for this line
* 3rd party integrations. eg. exchange or LDAP integration
